src/protocol/sort/multi_bit_permutation.rs

```diff
@@ -236,11 +235,15 @@ mod tests {
 
     use super::check_everything;
 
-    const INPUT: [&[u128]; 3] = [
-        &[0, 0, 1, 0, 1, 0],
-        &[0, 1, 1, 0, 0, 0],
-        &[1, 0, 1, 0, 1, 0],
+    const INPUT: [[u128; 3]; 6] = [
+        [0, 0, 1],
+        [0, 1, 0],
+        [1, 1, 1],
+        [0, 0, 0],
+        [1, 0, 1],
+        [0, 0, 0],
```

Comment:
Can you add a comment above this saying "This is in Little-Endian format"? I gets me every time.